common/settings.py

```diff
@@ -0,0 +1,69 @@
+import os
+import logging
+import logging.config as logging_config
+import sys
+
+__author__ = 'cnishina'
+
+linux = {
+    'base_dir': '/opt/alt',
+    'conf_dir': '/etc/alt/conf',
+    'log_conf': '/var/log/',
+}
+
+darwin = {}
+
+windows = {}
+
+
+class Settings:
+
+    def __init__(self):
+        self.env = 'dev'
+        self.run_module = None
+        self.base_dir = None
+        self.conf_dir = None
+        self.log_conf = None
+
+        self.db_config = None
+        self.alt_config = None
+        self.proxy_config = None
+
+    # load the environment settings, default is to run as 'dev'
+    def load_env(self, env):
+        self.env = env
+        if self.env == 'dev':
+            self.base_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+            self.conf_dir = os.path.join(self.base_dir, 'conf')
+            self.log_conf = os.path.join(self.conf_dir, 'logging.conf')
+        else:
+            os_setting = None
+            if 'linux' in sys.platform:
+                os_setting = linux
+            elif 'darwin' in sys.platform:
+                os_setting = linux
+            elif 'win32' in sys.platform:
+                print 'windows'
```

Comment:
throw an exception to tell the user that its not supported yet? what os does OSX resolve to?